packages/cli/src/commands/releasegold/refund-and-finalize.ts

```diff
@@ -0,0 +1,29 @@
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { ReleaseGoldCommand } from './release-gold'
+
+export default class RefundAndFinalize extends ReleaseGoldCommand {
+  static description =
+    "Refund the given contract's balance to the appopriate parties and destroy the contact"
```

Comment:
Consider including `Can only be called by the release owner of revokable ReleaseGold instances.`, or something to make it clear to users that they are unlikely to need this command